packages/core/src/utils/fuzzy.ts

```diff
@@ -20,9 +20,10 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 import { match, surround } from "fuzzyjs";
 import { clone } from "./clone";
 
-export function fuzzy<T extends { id: string }>(
+export function fuzzy<T>(
   query: string,
   items: T[],
+  getIdentifier: (item: T) => string,
```

Comment:
The first bug highlighted here https://github.com/streetwriters/notesnook/issues/7666 happens because the `id` of recent and default commands match, and it gets incorrectly identified as the same command by `fuzzy`. I've added a `getIdentifier` param to fix this. Not sure if this is the best fix